pom.xml

```diff
@@ -16,8 +16,8 @@
 
 	<modules>
 
-		<module>dependency-analyzer-backend</module>
-		<module>dependency-analyzer-communication</module>
-		<module>dependency-analyzer-rest</module>
+		<module>backend</module>
```

Comment:
I'd say that is much about the preference ;-) . There will be no client, so I wouldn't named it as a server. But it is just my preference :-)